tests/test_cookies.py

```diff
@@ -46,7 +48,83 @@ def test_cookies_bake(testdir):
     repo = template.ensure('{{cookiecutter.repo_name}}', dir=True)
     repo.join('README.rst').write(template_readme)
 
+    return template
+
+
+def test_cookies_bake_with_template_kwarg(testdir, cookiecutter_template):
+    """bake accepts a template kwarg."""
+    print(cookiecutter_template)
+    assert cookiecutter_template.check(dir=1, exists=1)
+
     testdir.makepyfile("""
+        # -*- coding: utf-8 -*-
+
+        def test_bake_project(cookies):
+            result = cookies.bake(
+                extra_context={'repo_name': 'helloworld'},
+                template=u'%s',
```

Comment:
Path on Windows is not properly escaped.To fix the error on Windows: s/u'%s'/r'%s'